trailblazer/store/api.py

```diff
@@ -246,8 +246,8 @@ def query_slurm(job_id_file: str, case_id: str, ssh: bool) -> Any:
         case_id: Unique internal case identifier which is expected to by the only item in the .YAML dict
         ssh : Whether the request is executed from hasta or clinical-db"""
         job_id_dict = safe_load(open(job_id_file))
-        submitted_jobs = job_id_dict[case_id]
-        jobs_string = ",".join(submitted_jobs)
+        submitted_jobs = job_id_dict.get(next(iter(job_id_dict)))
+        jobs_string = ",".join(map(str, submitted_jobs))
```

Comment:
```suggestion
        submitted_job_ids = job_id_dict.get(next(iter(job_id_dict)))
        job_ids_string = ",".join(map(str, submitted_job_ids))
```

Reply:
I think you should put L248-250 in a "get_submitted_job_ids" def in perhaps a `proccess` or `slurm` module or similar. It does not feel like this processing belongs in store/api and also you would get the benefit of writing a unit test to check this parsing. Also, the hardcoded input to check_ouput() looks like it belongs in a slurm_api. What do you think? Perhaps out of scope for this PR.

Reply:
Sure! This could be something for gRPC-based Trailblazer v.3 :D